bgpd/bgp_vty.c

```diff
@@ -771,18 +771,21 @@ static void bgp_clear_star_soft_out(struct vty *vty, const char *name)
 #endif
 
 /* BGP global configuration.  */
-
-DEFUN (bgp_multiple_instance_func,
-       bgp_multiple_instance_cmd,
-       "bgp multiple-instance",
-       BGP_STR
-       "Enable bgp multiple instance\n")
+#if defined(VERSION_TYPE_DEV) && (CONFDATE > 20190601)
```

Comment:
I thought we agreed to do revision based deprecation moving forward as to not have to go back and make changes to released code...

Reply:
Now I am confused this is based upon the work you did to clean this up.  What are you actually expecting to test against here?

Reply:
you're right - I had proposed making it version dependent and in discussion we decided to do just if dev or not -- defined(VERSION_TYPE_DEV).  This is fine.